### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#273] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152318158-fe26a570-4ccc-4bed-8478-cc6ce81244c1.png)

**Comment by neoSnakex34** [comment]
![ex](https://user-images.githubusercontent.com/97003970/152639027-8b9a912c-adbf-4e04-909c-2e021fbf2de2.jpg)

**Comment by Exyss** [comment]
Soluzioni analoghe a @neoSnakex34

**Comment by bertolinisapienza** [comment]
> ![ex](https://user-images.githubusercontent.com/97003970/152639027-8b9a912c-adbf-4e04-909c-2e021fbf2de2.jpg)

mi sa che te sei sbagliato, R^2 = R unito {(1,5),(3,5)}
